### PR TITLE
add release notes of BCD 1.0.1

### PR DESCRIPTION
**Pull request by JeremJR**

### CODE REVIEW

File: md/release_notes.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Release notes
+
+## Bug fixes
+
+### Fixes in BCD 1.0.1
+
```
Comment (tnguyen1) [inline md/release_notes.md]: What about this bug fix? https://bonitasoft.atlassian.net/browse/BCD-110
Have you omit it intentionnaly?

File: md/taxonomy.md [hunk 2]
```diff
@@ -9,3 +9,4 @@
 * [BCD Controller Docker image](_docker.md)
     * [Docker image overview](docker.md)
     * [BCD Command Line Interface](cli.md)
+* [Release notes](release_notes.md)
```
Comment (tnguyen1) [inline md/taxonomy.md]: For Bonita, the "Release Notes" link is the first in the navigation tree.
Maybe we should remain consistent?

File: md/release_notes.md (hunk 1 above)
Comment (tnguyen1) [reply md/release_notes.md]: OK to exclude this bug fix as it deals with the `lic_request` parameter that we don't want to advertise.

File: md/taxonomy.md (hunk 2 above)
Comment (JeremJR) [reply md/taxonomy.md]: I am agree, this is the fix https://github.com/bonitasoft/bonita-continuous-delivery-doc/pull/25/commits/2754ad11b01d0c576fc6608c49aef12bb06e8df0

**Pull request merged**